lib/query.js

```diff
@@ -56,12 +56,23 @@ export const isInteger = node => {
 }
 
 export const isPolynomial = (node) => {
+    if (isPolynomialTerm(node)) {
+        return true
+    } else if (isApply(node)){
```

Comment:
I am basing most of this logic on: https://www.mathsisfun.com/algebra/polynomials.html. It says that constants and polynomialTerms alike are considered polynomials.